### ISSUE TITLE
bug - Splunk 9.4.0 binary naming convention changes lead to Automator failure

### ISSUE THREAD

**Issue by guilhemmarchand** [thesplunker/Splunkenizer#41]
**Splunk changes in the binary names**

With Splunk 9.4.0, binary names have changed, and became inconcistent, leading to the failure in identifying these:

```
splunk-9.4.0-6b4ebe426ca6-linux-amd64.tgz
splunkforwarder-9.4.0-6b4ebe426ca6-linux-amd64.tgz
```

```
10:32:18 [WARNING]:  * Failed to parse /home/jenkins/agent/splunk-platform-
10:32:18 automator/config/splunk_config.yml with auto plugin: Error: Linux Archive for
10:32:18 splunk version latest missing in /home/jenkins/agent/splunk-platform-
10:32:18 automator/../Software
```

**Comment by splunkenizer** [comment]
Thanks for submitting. Working on it already...

**Comment by splunkenizer** [comment]
This has been fixed with version 2.2.4
